src/mme-app/handlers/attach_csresp.c

```diff
@@ -178,6 +178,7 @@ post_to_next()
 	/*Call DUMMY MB funcion*/
 	//test: send_dummy_mbr();
 	attach_stage6_counter++;
+	log_msg(LOG_INFO, "Post for s1ap processing - stage 6. SUCCESS\n");
```

Comment:
For now its ok to have stage6...But Would be really good if we write message name which is successfully processed.. eg. Create Session Response